validator/project_validator.go

```diff
@@ -1912,8 +1913,8 @@ func checkTaskTimeout(project *model.Project) ValidationErrors {
 			errs = append(errs,
 				ValidationError{
 					Message: fmt.Sprintf("project '%s' does not "+
-						"have an exec_timeout_secs defined on one or more tasks; these tasks will default to a timeout of 6 hours",
-						project.Identifier),
+						"have an exec_timeout_secs defined at the top-level or on one or more tasks; these tasks will default to a timeout of %d hours",
+						project.Identifier, agent.DefaultExecTimeout/time.Hour),
```

Comment:
Instead of dividing by time.Hour, can we use the .Hours() function and then convert to an int?

Reply:
oh okay